_assets/less/application.less

```diff
@@ -1,109 +0,0 @@
-/* Styles applied site-wide. */
-
-footer {
-	margin-bottom: 1em;
-}
-
-h1 {
-	margin-top: 0.2em;
-}
-h2 {
-	margin-bottom: 15px;
-}
```

Comment:
This could actually be the reason for differences on Latest News header btw @vadz .

I pulled these custom margins out of the styles when switching to BS4, so I could see if defaults were adequate now. Maybe they aren't, or maybe they just aren't for that homepage?